app/models/like.rb

```diff
@@ -0,0 +1,11 @@
+class Like < ApplicationRecord
+  belongs_to :post, :user
```

Comment:
- [ ] <b>REQUIRED:</b> We will be needing to access the user and an `author`, so it is recommended to define it as such.

```ruby
belongs_to :author, class_name: 'User'
```

Reply:
This change has been done perfectly